.github/workflows/wit-wasmcloud-lattice-bus.yml

```diff
@@ -0,0 +1,36 @@
+name: wit-wasmcloud-bus-publish
+
+on:
+  push:
+    tags:
+      - 'wit-wasmcloud-bus-v*'
+
+permissions:
+  contents: read
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+        sparse-checkout: |
+          wit
+    - name: Extract tag context
+      id: ctx
+      run: |
+          version=${GITHUB_REF_NAME#wit-wasmcloud-secrets-v}
+          echo "version=${version}" >> "$GITHUB_OUTPUT"
+          echo "tarball=wit-wasmcloud-secrets-${version}.tar.gz" >> "$GITHUB_OUTPUT"
+          echo "version is ${version}"
+    - name: Build
+      run: |
+        tar cvzf ${{ steps.ctx.outputs.tarball }} -C wit secrets/wit
```

Comment:
```suggestion
        tar -cvzf ${{ steps.ctx.outputs.tarball }} -C wit secrets/wit
```

Reply:
Thanks! Caught a real bug with my copy-paste. Weirdly tar is a-ok with not passing in a hyphen here. I adjusted the other workflows I copied from to include this as well.

Reply:
There's no bug here. `-` is not required by either GNU nor BSD tar (i.e. neither on Linux, nor on Mac by default), "hyphen-less" `tar` usage is actually much more common